### PR TITLE
[v6r17] Improve pretty printing of tables

### PR DESCRIPTION
**Pull request by phicharp**
The problem was that if a field is too long, all lines would be padded with the same length of blanks. Now the last field may wrap lines and no screw up the table.

Try for example 
dirac-admin-show-task-queues --task 22215364 -v

see pictures below

### CODE REVIEW

**Comment by phicharp** [comment]
new
old

**Comment by phicharp** [comment]
In blue is the new table (top), in back is the old table (bottom)

File: Core/Utilities/PrettyPrint.py [hunk 1]
```diff
@@ -18,95 +18,91 @@ def int_with_commas( inputValue ):
...
-    lengths.append( len( fieldList[i] ) )
-    for r in recordList:
-      r[i] = r[i].strip()
-      if len( r[i] ) > lengths[i]:
-        lengths[i] = len( r[i] )
-
+  # Compute the maximum width for each field
+  fieldWidths = []
+  integerField = []
+  for i in range( nFields ):
```
Comment (fstagni) [inline Core/Utilities/PrettyPrint.py]:
```
for inF in xrange(nFields):
```

**Comment by atsareg** [comment]
The result does not look like a table. What can be better is to print site names as a column, one site per line. The output will be longer but much more readable

**Comment by atsareg** [comment]
I mean something like that (with a minor change in the command script):

   $ dirac-admin-show-task-queues --task 22215364 -v
    
   ==> TQ 22215364
   Key        Value 
   ==================================================================================================
   CPUTime                                                                                     86400
   JobTypes   MCSimulation
   Jobs                                                                                          638
   OwnerDN    /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=charpent/CN=381668/CN=Philippe Charpentier
   OwnerGroup lhcb_mc
   Platforms  x86_64-slc6-gcc49-opt
              x86_64-slc6-gcc48-opt
              x86_64-slc6-gcc46-opt
              x86_64-slc6-gcc44-opt
              x86_64-slc6
   Priority   0.024
   Setup      LHCb-Production
   Sites      DIRAC.Test.ch
              VAC.BHAM-HEP.uk
              LCG.Legnaro.it
              DIRAC.Zurich.ch
              LCG.IFJ-PAN.pl
              LCG.PNPI.ru
              ...

**Comment by atsareg** [comment]
screen shot 2017-03-15 at 21 09 45

**Comment by phicharp** [comment]
Close as superseded by another PR from Andrei

**Pull request closed without merging**